tcms/testplans/tests/tests.py

```diff
@@ -66,76 +66,14 @@ def setUp(self):
         self.client.login(username=self.user.username,  # nosec:B106:hardcoded_password_funcarg
                           password='admin')
 
+
+class PlanTests(BasePlanTest):
+
     def test_open_plans_search(self):
         location = reverse('plans-search')
         response = self.client.get(location)
         self.assertEqual(response.status_code, HTTPStatus.OK)
 
-    def test_plan_new_get(self):
```

Comment:
Do not delete something which is added in the next commit. Moving stuff around needs to be an atomic operation so that if, for whatever reason we decide to go back to exactly this commit, we are not losing existing functionality and everything still works. (at least tests are passing).

So to fix this you have to bring back the deleted lines in this commit and the only change that will be left here is splitting the test class into 2 other classes.

Then in the next commit you will delete these methods from here and add them in the new file (which is already done). In this way the current commit will show the preparation step and the next one will contain the actual change.